tests/rptest/tests/replication_factor_change_test.py

```diff
@@ -56,20 +55,14 @@ def wait_rec():
                                      self.replication_factor)
         self.check_rf(self.replication_factor)
 
-    @cluster(num_nodes=4, log_allow_list=CHANGE_REPLICATION_FACTOR_ALLOW_LIST)
+    @cluster(num_nodes=4)
     def check_error_test(self):
         self.replication_factor = -1
-        try:
-            self._rpk.alter_topic_config(self.topic_name, self.rf_property,
-                                         self.replication_factor)
-            assert False, "We can not set replication factor <= 0"
-        except:
-            assert len(self.admin.list_reconfigurations()) == 0
+        res = self._rpk.alter_topic_config(self.topic_name, self.rf_property,
```

Comment:
Is there an issue open for rpk returning 0 in this case?  If not we should open one

Reply:
I think, not
Let me do it